packages/fxa-auth-server/lib/payments/configuration/manager.ts

```diff
@@ -1,143 +1,40 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-import { Firestore } from '@google-cloud/firestore';
 import { randomUUID } from 'crypto';
 import { Logger } from 'mozlog';
 import { Container } from 'typedi';
-import { TypedCollectionReference } from 'typesafe-node-firestore';
 
 import errors from '../../error';
 import { AppConfig, AuthFirestore, AuthLogger } from '../../types';
+
+import {
+  PaymentConfigManager as PaymentConfigManagerBase,
```

Comment:
I kept the class names in fxa-shared the same and then do a 'as' here. Not sure if this good style or not. Let me know your thoughts. Happy to change this if needed.